CONTRIBUTING.md

```diff
@@ -0,0 +1,231 @@
+<!-- omit in toc -->
+# Contributing
+
+- [Types of Contribution](#types-of-contribution)
+  - [Non-Programming](#non-programming)
+  - [Programming](#programming)
+- [Submitting Your Work](#submitting-your-work)
+- [Expectations Regarding Timelines](#expectations-regarding-timelines)
+- [Discuss \& Design First](#discuss--design-first)
+- [Getting the Source \& Installing](#getting-the-source--installing)
+  - [Install into a Virtual Python Environment](#install-into-a-virtual-python-environment)
+  - [Install in Editable Mode](#install-in-editable-mode)
+- [Command Design](#command-design)
+  - [Implementation Examples](#implementation-examples)
+  - [Helpers \& Utilities](#helpers--utilities)
+  - [Logging](#logging)
+  - [Code Documentation](#code-documentation)
+  - [Code Style](#code-style)
+- [Maintainers Notes](#maintainers-notes)
+  - [Release process](#release-process)
+- [Maintainership](#maintainership)
+
+
+## Types of Contribution
+
+No matter if you're a programmer or not there are many ways to contribute to the `synadm` project.
+
+### Non-Programming
+
+* [Set up synadm](README.md#installation) and report whether the installation, configuration and update processes work as described or could be improved. We don't have the resources to test on many operating systems, thus a quick GitHub issue telling us: "It even works on Fancy-OS" would help already.
```

Comment:
> We don't have the resources to test on many operating systems, thus a quick GitHub issue telling us: "It even works on Fancy-OS" would help already.

Not quite a good idea, it encourages to spam the issue tracker with non-issues. I think it's less worse if it's thrown into the room for synadm instead.

```suggestion
* [Set up synadm](README.md#installation) and report whether the installation, configuration and update processes work as described or could be improved. We don't have the resources to test on many operating systems, but you can [let us know](https://matrix.to/#/%23synadm%3Apeek-a-boo.at?via=jacksonchen666.com&via=maclemon.at&via=matrix.org&via=raim.ist) if it does work.
```

Reply:
Agree, I just had that in since the first versions where I was not sure if that damn thing works for anyone, so yeah we are past that long ago. Like your suggestion! Nice!